templates/ntp.conf.erb

```diff
@@ -36,12 +36,12 @@ server <%= server %><% if @server_options != 'UNSET' %> <%= @server_options %><%
 
 <% if @my_peers != ['UNSET'] -%>
 # Peers
-<%   if @peers_type == 'string' or @peers_type == 'array' -%>
+<%   if @my_peers.is_a?(Array) -%>
```

Comment:
not possible for this output variable, please see the input handling here: https://github.com/ghoneycutt/puppet-module-ntp/blob/master/manifests/init.pp#L69-L85